NitroxClient/Helpers/ThrottledPacketSender.cs

```diff
@@ -48,8 +48,12 @@ public bool SendThrottled(Packet packet, Func<Packet, object> dedupeMethod, floa
                 return true;
             }
 
+            if (!packetSender.SendIfGameCode(packet))
+            {
+                return false;
```

Comment:
Contract of this method isn't very clear.  In this case, the caller will assume that the packet was not sent due to throttling and may branch/do other logic based on this assumption. (worst case, tuck the event away to send later, which would not be wanted in this case).

Reply:
I have removed the bool return and changed it to an if..else because it had an either/or intend before.